docs/quest_template_locale.md

```diff
@@ -8,141 +8,68 @@ This table is used to provide to localized clients with localized string for que
-| [ObjectiveText3\_loc8](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc1](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc2](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc3](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc4](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc5](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc6](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc8](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
-| [ObjectiveText4\_loc8](#quest_template_locale-ObjectiveText_loc)      | text         | signed         |         | YES      | NULL        |           |             |
+| Field          | Type                  | Collation       | Null | Key | Default | Extra | Comment |
+|----------------|-----------------------|-----------------|------|-----|---------|-------|---------|
+| ID             | mediumint(8) unsigned | (NULL)          | NO   | PRI | 0       |       |         |
+| locale         | varchar(4)            | utf8_general_ci | NO   | PRI | (NULL)  |       |         |
+| Title          | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| Details        | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| Objectives     | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| EndText        | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| CompletedText  | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| ObjectiveText1 | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| ObjectiveText2 | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| ObjectiveText3 | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| ObjectiveText4 | text                  | utf8_general_ci | YES  |     | (NULL)  |       |         |
+| VerifiedBuild  | smallint(5)           | (NULL)          | YES  |     | 0       |       |         |
 
```

Comment:
Add the links again

Reply:
you do it easily like this:
```md
| Field | Type | Attribute | Key | Null | Default | Extra | Comment
:--- | :--- | :--- | :--- | :--- | :--- | :--- | :---
[Entry][1] | mediumint(8) | signed | PRI | | 0
[SchoolMask][2] | tinyint(4) | signed | | | 0
[SpellFamilyName][3] | smallint(5) | unsigned | | | 0
[SpellFamilyMask0][4] | int(10) | unsigned | | | 0
[SpellFamilyMask1][5] | int(10) | unsigned | | | 0
[SpellFamilyMask2][6] | int(10) | unsigned | | | 0
[procFlags][7] | int(10) | unsigned | | | 0
[procEx][8] | int(10) | unsigned | | | 0
[ppmRate][9] | float | signed | | | 0
[CustomChance][10] | float | signed | | | 0
[Cooldown][11] | int(10) | unsigned | | | 0

[1]: #entry
[2]: #schoolmask
[3]: #spellfamilyname
[4]: #spellfamilymask0
[5]: #spellfamilymask1
[6]: #spellfamilymask2
[7]: #procflags
[8]: #procex
[9]: #ppmrate
[10]: #customchance
[11]: #cooldown
```
remember you need it to be lower case, otherwise it wont work

Reply:
Interesting fact about the link, the truth was not sure how it worked.